index.html

```diff
@@ -24,6 +24,8 @@
       color: white;
     }
   </style>
+  <script src="scripts.js"> </script>
+  <script src="assignment.js"> </script>
```

Comment:
Since we already have assignment.js file inside 01-Fundamentals... , you can insert your changes right there and not create a new one for now.